tutorials/spa-create-forms/spa-create-forms.md

```diff
@@ -233,44 +233,65 @@ These notifications will appear in the inbox of the requester as a task.
 
-8. Add the new form to the rejection route of the approval form.
+8. To add the new rejection form , we will use the **Duplicate** feature.
 
-    !![03-027](unit3-30.png)
+    **Duplicate** feature is used to copy artifacts within the Business Process whereas **Copy** feature is used to copy artifacts across the business processes.
 
-9. In the **Create Form** dialog box, do the following:
-    - Enter **Order Rejection Form** in the **Name** field.
-    - Enter **Notification form to inform that the sales order is rejected by the supplier** in the **Description** field.
-    - Choose **Create**.
+    Click on the **Overview**.
+
+    !![Click Overview](Click-Overview.png)
+
+9. In the **Overview** screen, do the following:
+    - Find **Order Confirmation Notification** under the **Artifacts** section and click on **...**.
+    - Choose **Duplicate**.
+
+    !![Duplicate Form](DuplicateForm.png)
+
+11. The duplicate artifact pop-up will appear.
+
+    - !![Duplicate Artifact](DuplicateArtifact.png)
+
+    Change the name to **Order Rejection Notification** and click **Duplicate**.
+
+    - !![Order Rejection Form](NameOrderRejectionNotification.png)
+
+12. The **Order Rejection Notification** form is automatically opened in the form builder.
+
+    The screen shown will be same as **Order Confirmation Notification**.
 
-    !![03-026](unit3-31a.png)
+    - !![Order Rejection Form](DuplicatedForm.png)
 
-10. Design the order rejection form in the form builder.
+13. Change the order rejection form in the form builder to reflect the data for rejection case.
 
     - Do not forget the **Save** the form once completed.
 
     | Form Fields | Field Settings with Label
     |  :------------- | :-------------
     | Headline 1 | Order Rejection
     | Paragraph  |We are sorry to inform you that your order cannot not be accepted. Any inconvenience caused due to refusal of order is regretted. You can find the reason of rejection and the details of your order below, please confirm the request:
-    | Text Area  | Message from the supplier:
-    | Paragraph  | Your Sale's Order Details:
 
-    - For all below **Input Fields** enter the labels and select the **Read Only** checkbox.
+    - For the below **Input Field** enter the label and select the **Read Only** checkbox.
 
     | Form Fields| Field Settings with Label
     |  :------------- | :-------------
-    | Text | Order Number
-    | Number | Order Amount
-    | Date | Expected Delivery Date
     | Paragraph | Please press the SUBMIT button to acknowledge the order status.
 
-    !![Order Rejection](order-rejection.png)
+    !![Order Rejection](ChangeOrderRejectionNotification.png)
 
     - Do not forget the **Save** the form once completed.
 
-    !![Save Order Rejection](save-order-rejection.png)
+    !![Save Order Rejection](OrderRejectionNotificationAfterSave.png)
+
+    Rest all fields can be left as is.
```

Comment:
This sounds a bit odd. Did you want to say something like "The remaining fields can be left as is."?